### ISSUE TITLE
Unexpected token 'export' in nuxt.config.js

### ISSUE THREAD

**Issue by DiscreteTom** [michalzaq12/electron-nuxt#790]
Environment:
- UI framework: Vuetify
- css pre-processor: none
- icon set: none
- typescript: N
- ESLint: N

After `yarn install`, when `yarn dev`, this issue occurs:

```
xxx/src/render/nuxt.config.js:8
export default {
^^^^^^

SyntaxError: Unexpected token 'export'
```

In `src/renderer/nuxt.config.js`, change `export default` to `module.exports` can fix this issue.

**Comment by petertorelli** [comment]
Same problem, different environment:

? Select which ui-components framework install none
? Select which css pre-processor install Sass
? Select which icon set install Font Awesome 5
? Use typescript? Yes
? Use linting with ESLint? Yes

**Comment by michalzaq12** [comment]
Fixed in 1.8.1

**Comment by joeelia** [comment]
Yes, but now with modules.export you get errors when you do `yarn install`. If you change it to be `export default {` then it compiles with `yarn install`, but then you need to change it back to `modules.export` for it to work on `yarn dev`.

The solution is actually to modify the `.eslintrc.js` in root folder and add the following line under `rules: {`

Add this line:
`        'nuxt/no-cjs-in-config': 'off',`

Now it works for `yarn install` and `yarn dev`
